### PR TITLE
add of format_string module

### PR DESCRIPTION
**Pull request by xavitator**
Refer to issue #3.

### CODE REVIEW

**Comment by kolibs** [comment]
Everything is fine for me. Please, just rewrite your comment with odoc syntax and I will merge your pull request.
https://github.com/ocaml/odoc

File: src/lib/FString.ml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  let nb_blank = size - String.length content in
+  match pos with
+  | Center -> let space_after = nb_blank / 2 in 
+    (String.make (nb_blank - space_after) ' ') 
+    ^ content 
+    ^ (String.make space_after ' ')
+  | Left -> content ^ (String.make nb_blank ' ')
+  | Right -> (String.make nb_blank ' ') ^ content
+
+(** Main focntion to format string with these arguments :
```
Comment (kolibs) [inline src/lib/FString.ml]: typo

File: src/lib/FString.ml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+(** Module in charge of format string with a size and a position *)
+
+(** Type corresponding to a position *)
+
+type position = 
+  | Center
+  | Left
+  | Right 
+
+(** This fonction add space to the content, depending to the pos argument *)
```
Comment (kolibs) [inline src/lib/FString.ml]: typo *function*

File: src/lib/FString.mli [hunk 3]
```diff
@@ -0,0 +1,20 @@
+(** Module in charge of format string with a size and a position *)
+
+
+(** Type corresponding to a position *)
+type position = 
+  | Center
+  | Left
+  | Right 
+
+(** Main focntion to format string with these arguments :
```
Comment (kolibs) [inline src/lib/FString.mli]: Copy and paste booouhhh!
Same typo...

Comment (xavitator) [reply src/lib/FString.mli]: Syntax fault, bouuuh, you wrote "fucntion" instead of "function" 😂

**Pull request merged**